src/main/java/org/kuali/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -1468,6 +1474,13 @@ protected boolean isTaxReviewRequired() {
      * @return true if the payee is a vendor and has withholding dates therefore should receive tax review, false otherwise
      */
     protected boolean isPayeePurchaseOrderVendorHasWithholding() {
+        /* CU Customization: Null checks added to prevent NPE when payee is not selected on saved DV causing buttons to not present for a super user.
+         */
+        if (ObjectUtils.isNull(this.getDvPayeeDetail()) || 
```

Comment:
Can this overlay's extra null checks be removed? If the code can be updated so that DisbursementVoucherDocument and CuDisbursementVoucherDocument share the same "dvPayeeDetail" variable, the custom null checks might not be necessary.